### PR TITLE
[Stack 0] Update intended use document

### PR DESCRIPTION
**Pull request by SimonBiggs**
This is updating arguably one of the most important regulatory documents, the intended use.

After feedback from the TGA, my software, although it is clinical decision support software, it is not exempt. And therefore all wording suggesting its exemption has been removed.

Also, the software no longer runs with a cloud connection, and therefore all details around cloud requirements have been removed.

### CODE REVIEW

**Comment by SimonBiggs** [comment]
Also, I was looking to pull in the definition of `Contourer` from http://ihe-ro.org/doku.php?id=doc:profiles:brto-ii:

> **Contourer** – A system that consumes one or more CT image series and creates an RT Structure Set. If the Contourer consumes multiple CT image series or has an internal requirement for resampling, it also will generate a single CT image series to which the RT Structure Set maps. A Contourer shall be able to consume CT image series with non-uniform spacing.

But, unfortunately it explicitly states "CT image series". I wonder why it doesn't just say "image series". Gone are the days where it is an expectation that a CT is used at all (with a TPS potentially being able to create synthetic CT images from MRI images for dosimetric calculation and all that...).

**Comment by sjswerdloff** [comment]
>I wonder why it doesn't just say "image series".

The purpose was interoperability of systems.  At the time (>15 years ago), it was difficult enough to ensure that an RT SS coming from one system could be consumed by another.
The definition was to a specific actor in a specific context.

But your point is well made regarding it being a bit of an anachronism now, and that is something you could bring up with the IHE-RO Technical committee (although they would have to provide a different name for the thing that would have your preferred definition assigned).

However, I have to say that I still think that X-Ray CT is a better modality for providing stopping powers (through calibration and calculation) than other modalities (I think it's a significant additional approximation to get the information from MR).
There might be a different actor that provides Contouring on other modalities (in conjunction with registration/SRO on the CT that one might expect to be used by a dose calculation system).

File: docs/0-intended-use/index.md [hunk 1]
```diff
@@ -219,53 +170,21 @@ specific servers.
...
-
-Only identifying patient information that is stored within the standards
-compliant DICOM header is pseudonymised before sending to the cloud sever.
-
-The DICOM software can be configured to only send through certain DICOM file
-types. By default this is CT, MRI and RT-DICOM specific files. This is to help
-prevent bulk DICOM sends containing other DICOM types that sometimes have burnt
-in pixel data included having those extra images included in the
-pseudonymisation + upload.
+standard DICOM input output (either file based or DICOM network protocol
```
Comment (sjswerdloff) [inline docs/0-intended-use/index.md:186]: input and/or output?

File: docs/0-intended-use/index.md [hunk 2]
```diff
@@ -278,11 +197,11 @@ configurable alerts and warnings can provide the appropriate notification.
 ## Part of the Body / Type of Tissue Interacted With
 
 The device is stand-alone software. It receives input from the user and outputs
```
Comment (sjswerdloff) [inline docs/0-intended-use/index.md:199]: What input does it receive from the user (if the data comes over the network)?
It also receives input from the network if the DICOM DIMSE/wire protocol is in use

**Review by sjswerdloff — APPROVED** [review]

File: docs/0-intended-use/index.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/0-intended-use/index.md]: Fixed in d9fc69e62dcf8ed6eca258b7beb0e20c32bc031e

File: docs/0-intended-use/index.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/0-intended-use/index.md]: Fixed in d9fc69e62dcf8ed6eca258b7beb0e20c32bc031e

**Comment by SimonBiggs** [comment]
Unfortunately, due to a merge conflict this PR approval dropped.

Only extra changes were to fix the two comments, fix the merge conflict, and update the cache version within the CI workflow

**Review by sjswerdloff — APPROVED** [review]

**Pull request merged**